src/components/AlternatingContent.js

```diff
@@ -3,14 +3,17 @@ import PropTypes from 'prop-types'
 import AlternatingContentSection from './AlternatingContentSection'
 import contentBlockPropTypes from './TwoColumnContentBlock/contentBlockPropTypes'
 import TwoColumnContentBlock from './TwoColumnContentBlock/TwoColumnContentBlock'
+import Section from './Section/Section'
 
 const AlternatingContent = ({ contentItems }) =>
   contentItems && !!contentItems.length ? (
-    <AlternatingContentSection>
-      {contentItems.map((item, index) => (
-        <TwoColumnContentBlock {...item} key={index} />
-      ))}
-    </AlternatingContentSection>
+    <Section>
```

Comment:
Additional DOM for no good reason

Reply:
This is basically Bulma DOM to use the opportunities from there